### DISCUSSION TITLE
@Recurring job continues to run even though I comment it out

### DISCUSSION THREAD

**Discussion by bsanders1979** [jobrunr/jobrunr#812] [General]
Hello,
I use Spring Boot and have a method annotated with @Recurring. I didn't want it to run so I commented out the annotation. However, the job keeps running. I see an entry in the recurring jobs table, so I'm guessing that the lack of annotation is essentially being ignored. How can I disable it?

**Comment by rdehuyss** [comment]
Delete it via the dashboard...

I cannot just delete recurring jobs as some may have been created with an API call using `jobScheduler.scheduleRecurringly` and you may want not to have those deleted automatically.
